org.eclipse.winery.repository.ui/src/app/instance/instance.service.ts

```diff
@@ -44,7 +44,7 @@ export class InstanceService {
 
         switch (this.toscaComponent.toscaType) {
             case ToscaTypes.NodeType:
-                subMenu = ['README', 'LICENSE', 'Visual Appearance', 'Instance States', 'Interfaces', 'Implementations',
```

Comment:
all definition childs get the tag menu, but only the nodeType gets the Tag module? Is this intended? What about service templates?

![grafik](https://user-images.githubusercontent.com/1366654/42280237-715876e2-7fa0-11e8-89e2-0853bc3ce373.png)

(Taken from http://docs.oasis-open.org/tosca/TOSCA/v1.0/TOSCA-v1.0.html)

Reply:
Service Templates already have Tags: https://github.com/eclipse/winery/blob/90a7962693683c550cf4533389e0235af908abea/org.eclipse.winery.repository.ui/src/app/wineryMainModules/serviceTemplates/serviceTemplateRouter.module.ts#L63 and https://github.com/eclipse/winery/blob/90a7962693683c550cf4533389e0235af908abea/org.eclipse.winery.repository.ui/src/app/instance/instance.service.ts#L53